include/class.banlist.php

```diff
@@ -25,8 +25,31 @@ function remove($email) {
         return self::getSystemBanList()->removeRule('email','equal',$email);
     }
 
-    function isbanned($email) {
-        return TicketFilter::isBanned($email);
+    /**
+     * Quick function to determine if the received email-address is in the
+     * banlist. Returns the filter of the filter that has the address
+     * blacklisted and FALSE if the email is not blacklisted.
+     *
+     */
+    static function isBanned($addr) {
+
+        if (!($filter=self::getFilter()))
+            return false;
+
+        $sql='SELECT filter.id '
+            .' FROM '.FILTER_TABLE.' filter'
+            .' INNER JOIN '.FILTER_RULE_TABLE.' rule'
+            .'  ON (filter.id=rule.filter_id)'
+            .' WHERE filter.id='.db_input($filter->getId())
+            .'   AND filter.isactive'
```

Comment:
Should we add a check here to only find rules attached to the `SYSTEM BAN LIST` filter?